### PR TITLE
Add formula for fifechan 0.1.2

### PR DESCRIPTION
**Pull request by stubb**
Adds a formula for the fifechan library.

### CODE REVIEW

File: Library/Formula/fifechan.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Fifechan < Formula
+  desc "Is a C++ GUI library designed for games."
```
Comment (bfontaine) [inline Library/Formula/fifechan.rb]: You can remove “`Is a`”.

**Comment by dunn** [comment]
If this is _only_ for games, I wonder if we should have it in our Games tap instead.

**Comment by mikemcquaid** [comment]
> If this is only for games, I wonder if we should have it in our Games tap instead.

Agreed :+1:.

**Comment by stubb** [comment]
What do you mean by "Games tab"?

**Comment by stubb** [comment]
Nervermind. Figured it out https://github.com/Homebrew/homebrew-games/pull/457

**Pull request closed without merging**